htdocs/core/lib/project.lib.php

```diff
@@ -1124,9 +1124,9 @@ function projectLinesPerAction(&$inc, $parent, $fuser, $lines, &$level, &$projec
 			$projectstatic->ref = $lines[$i]->project_ref;
 			$projectstatic->title = $lines[$i]->project_label;
 			$projectstatic->public = $lines[$i]->public;
-			$projectstatic->status = $lines[$i]->project_status;
+			$projectstatic->status = $lines[$i]->project->status;
```

Comment:
When checking if the $lines[$i]->project->status; is correctly set to validate this change i have discovered that the full method  projectLinesPerAction()  is used nowhere into Dolibarr code. So this means this method is dead code and can be completely removed.

Reply:
done